src/renderer/views/Pins.js

```diff
@@ -232,6 +287,49 @@ class Pins extends Component {
                     {rows}
                 </tbody>
             </Table>
+            <h6>Select to pin and help secure the network by backing up videos</h6>
+            <input type='text' placeholder='Enter community ID...' onChange={(event) => {
+                if (event.target.value.match(/\bhive-\d{6}\b/g)) {
+                    this.updateSearchTables(event.target.value)
+                }
+            }} />
+            <Row>
+                {['new', 'trending'].map(type => (
+                    <Col>
+                        <Table striped bordered hover size='sm'>
+                            <thead>
+                            <tr>
+                                <th>{type} videos</th>
+                                <th>Title</th>
+                                <th>Creator</th>
+                                <th>pinned</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            {this.state[`${type}Videos`].map(video => (
+                                <tr>
+                                    <td><img src={video.images.thumbnail} /></td>
+                                    <td>{video.title}</td>
+                                    <td>{video.author}</td>
+                                    <td>{video.isPinned ? (
+                                        <Button variant="danger" onClick={async() => {
+                                            await this.removePin(video.id)
+                                            this.updateSearchTables()
+                                        }}>X</Button>
+                                    ) : (
+                                        <button variant="success" onClick={async() => {
```

Comment:
Would probably be best to use the same type of button (react-bootstrap) vs using default HTML button styling. Example: `<Button variant="success">`